osquery/tables/system/windows/chassis_info.cpp

```diff
@@ -33,11 +33,21 @@ QueryData genChassisInfo(QueryContext& context) {
     Row r;
     auto isPresent = false;
     long number;
+    std::vector<long> chassisTypes;
     data.GetBool("AudibleAlarm", isPresent);
     r["audible_alarm"] = isPresent ? "True" : "False";
     data.GetString("BreachDescription", r["breach_description"]);
-    data.GetLong("ChassisTypes", number);
-    r["chassis_types"] = INTEGER(number);
+    data.GetVectorOfLongs("ChassisTypes", chassisTypes);
+
+    std::ostringstream oValueConcat;
+    for (size_t i = 0; i < chassisTypes.size(); ++i) {
+      if (i != 0) {
+        oValueConcat << ",";
```

Comment:
nitpick, do you want this to read `1,2,3,4` or `1, 2, 3, 4`? If you do not want the space maybe you can `<< ','`.